### ISSUE TITLE
Not handling files with CRLF

### ISSUE THREAD

**Issue by gitonthescene** [orgapp/orgajs#46]
Parsing [this example](https://raw.githubusercontent.com/higham/org-mode-syntax-cheat-sheet/master/cheat_sheet.org) produces odd results.  I think changing [this split](https://github.com/orgapp/orgajs/blob/master/packages/orga/src/parser.ts#L38) to be `/\r\n|\n/` fixes this issue.

**Comment by xiaoxinghu** [comment]
@gitonthescene I suspect that this should be fixed in v2 (kind of 😄 ). Possible issue is that the `\r` is potentially being contained by the token prior to the `\r\n`. But I think it's mostly OK? Can you try it in the new version? Thanks.

**Comment by gitonthescene** [comment]
Thanks.  It definitely parses that example now, though I don't see any mention of `\r` in the code anywhere.  I'll let you know if I notice any issues with this.

**Comment by xiaoxinghu** [comment]
yes, [here](https://github.com/orgapp/orgajs/blob/8227e5bd3c4f457f69999483968521211993e266/packages/text-kit/src/read.ts#L15-L19) is the logic for breaking text into lines, it's a different way of maintaining the line positions since now we have to keep the `Position` info in our nodes. IMHO, it's probably not going to do any significant harm right now. 🤞
